terraform/cp4ba/terraform.tfvars.vpc

```diff
@@ -0,0 +1,43 @@
+// Openshift cluster parameters
```

Comment:
If you leave this file in here, be sure to add all of the variables for all of the other modules that can be customized.  This just has the `roks` parameters